src/admin/index.spec.js

```diff
@@ -66,7 +66,6 @@ describe('Admin', () => {
     const emitter = new InstrumentationEventEmitter()
     const cluster = createCluster({
       requestTimeout: 1,
-      enforceRequestTimeout: true,
```

Comment:
I don't think we should remove the `enforceRequestTimeout` option, since that would be a breaking change. Let's start by making it work, enabling it ourselves, and then at some point make it active by default.